flask-mongo/README.md

```diff
@@ -380,8 +356,10 @@ Depending on your OS, choose your adventure:
     reqTimestampMock: 2023-11-13T13:02:56.385067848Z
     resTimestampMock: 2023-11-13T13:02:56.386374941Z
   ```
+  
+  On terminal we can see the testcases generated -
 
-  Want to see if everything works as expected?
+  ![keploy testcases](./img/testcases.png)
```

Comment:
testCases added in the repo are failing due noise of python version.
<img width="1440" alt="Screenshot 2024-07-26 at 4 04 00 PM" src="https://github.com/user-attachments/assets/36bd41d2-c5da-4d58-b88b-9c405db95609">